src/screenshot/screenshot.c

```diff
@@ -54,9 +181,25 @@ static int method_screenshot(sd_bus_message *msg, void *data,
 		return -ENOMEM;
 	}
 
+	char *coordinates;
+	if (interactive) {
+		coordinates = exec_coordinates_selector();
+		if (coordinates == NULL || strlen(coordinates) == 0) {
```

Comment:
Nit: use `coordinates[0] != '\0'` or `!coordinates[0]` instead of `strlen(coordinates) == 0`